### PR TITLE
Branch find 2

### LINKED ISSUE #65: Update error message in find command

### LINKED ISSUE #69: Add implementation section for find feature to DG

### PR DESCRIPTION
**Pull request by dgc5213**
Edit Find command, DG, UG. 
closes #65 , closes #69

### CODE REVIEW

**Comment by dgc5213** [comment]
Hi @e0260222 , Please help merge it . thanks

File: docs/UserGuide.md [hunk 1]
```diff
@@ -135,7 +135,7 @@ Example of usage:
 
 Finds employee records that have any of the given keywords in their name/role.
 
-Format: `find TYPE/KEYWORD...`
+Format: `find TYPE/KEYWORD…`
 * `TYPE` must be the letter `n` or `r`.
 * If `TYPE` is `n`, the search will be based on name `[n/NAME]`.
```
Comment (wz-chan) [inline docs/UserGuide.md]: Perhaps `[n/NAME]` should be removed?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -135,7 +135,7 @@ Example of usage:
...
 * If `TYPE` is `r`, the search will be based on role `[r/ROLE]`.
```
Comment (wz-chan) [inline docs/UserGuide.md]: Perhaps `[r/ROLE]` should be removed?

**Review by wz-chan — CHANGES_REQUESTED** [review]
Your changes look great! Perhaps you could address the changes that I've suggested

**Review by wz-chan — APPROVED** [review]
LGTM

**Pull request merged**